x-pack/plugins/ml/public/application/model_management/model_actions.tsx

```diff
@@ -329,7 +329,11 @@ export function useModelActions({
         available: (item) =>
           item.model_type === TRAINED_MODEL_TYPE.PYTORCH &&
           canStartStopTrainedModels &&
-          (item.state === MODEL_STATE.STARTED || item.state === MODEL_STATE.STARTING),
+          (item.state === MODEL_STATE.STARTED || item.state === MODEL_STATE.STARTING) &&
+          // Only show the action if there is at least one deployment that is not used by the inference service
+          item.deployment_ids.some(
+            (dId) => !item.inference_apis.some((inference) => inference.model_id === dId)
```

Comment:
`inference_apis` is optional, I get a page crash when it doesn't exist.

Reply:
Yes just pushed the fix https://github.com/elastic/kibana/pull/175880/commits/ed91aca1e7f8aedafd0823f6311d60ceb02c9fb1,  for some reason TS check isn't working in my IDE 😕